src/components/MainActionButton.tsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable prettier/prettier */
+import React from 'react';
+import styled from 'styled-components';
+
+export interface mainActionButtonInterface {
+	text: string;
+	onClick: () => void;
+}
+
+const MainButton = styled.button`
+	background-color: #4961e1;
+	color: white;
+	font-family: Montserrat, Arial;
```

Comment:
We can remove this since it will be defined globally.

Reply:
By remove this, do you mean remove all the code that creates the styled component, or just certain lines in the styled component?

Reply:
Sorry, I meant the `font-family` only.